ios/RNDemo/NativeMobileSDKBridge.m

```diff
@@ -142,6 +142,19 @@ @implementation NativeMobileSDKBridge
 }
 
 #pragma mark: Media Related Function
+-(void)configureActiveAudioDevice:(NSArray<MediaDevice *> * _Nonnull)audioDeviceList
+{ 
+  // If wiredHeadphone connected, use Build-in receiver
+  // https://github.com/aws/amazon-chime-sdk-ios/blob/master/AmazonChimeSDK/AmazonChimeSDK/device/DefaultDeviceController.swift#L37
+  AVAudioSessionPortDescription *currentOutput = (AVAudioSessionPortDescription *)[[[[AVAudioSession sharedInstance] currentRoute] outputs] firstObject];
```

Comment:
Could you explain the logic for this?

Reply:
sry, I found rough in my implement.
I refactored and added commets.